src/vm/amd64/profiler.cpp

```diff
@@ -41,6 +41,11 @@ typedef struct _PROFILE_PLATFORM_SPECIFIC_DATA
     UINT64      flt5;
     UINT64      flt6;
     UINT64      flt7;
+    UINT64      rdi;
```

Comment:
rsi is here 
 mov                     [rsp + 0x20], rsi    //                 -- struct profiledRsp field